Docs fixes based on customer feedback (#1251)
telerik/blazor-docs: components/gridlayout/items.md

## ColumnSpan

The `ColumnSpan` parameter defines the how many columns the item will occupy.

> When you specify the desired `ColumnSpan`, you should also provide the `Column` that will serve as a start position for the item taking several columns.

Reviewer: "provide the `Column`" sounds ambiguous and I had to read it several times and guess what it means.

```suggestion
When you set the desired `ColumnSpan`, you should also set the `Column` parameter to specify the start position for the spanned item.
```

</TelerikGridLayout>

@code {

>     public int? ColumnSpan { get; set; } = 1;

Author: ```suggestion
    private int? ColumnSpan { get; set; } = 1;
```

}
````

## RowSpan

The `RowSpan` parameter defines how many rows the item will occupy.

> When you specify the desired `RowSpan`, you should also provide the `Row` that will serve as a start position for the item taking several rows.

Author: ```suggestion
When you set the desired `RowSpan`, you should also set the `Row` parameter to specify the start position for the spanned item.
```

</TelerikGridLayout>

@code {

>     public int? RowSpan { get; set; } = 1;

Author: ```suggestion
    private int? RowSpan { get; set; } = 1;
```
